hw_3/task3.js

```diff
@@ -0,0 +1,110 @@
+//«а»   «у»  «о»  «и»  й    «э»  «ю» «я»   ы     «е»  «ё»
+//1072 1091 1086 1080 1081 1101 1102 1102 1099  1077  1105
+//[а,у,о,и,й,э,ю,я,ы,е,ё]
+
+const vowels = "ауоийэюяыеёaeiou";
+
+const letterParser = (inputString) => {
```

Comment:
Create a string that contains consonants and add validation to it to handle special characters or numbers